### ISSUE TITLE
What's the function of VNET?

### ISSUE THREAD

**Issue by Chauncey-Wang** [GGchen1997/GDW-NIPS2021#1]
I am reading your great work, but I am confused about the vnet in your code. Could you explain the function of vnet in your code？

I guess `v_lambda[index, targets] = (torch.sum(v_lambda * p.data, dim=1) - v_lambda[index, targets] * p.data[index, targets]) / (1 - p.data[index, targets])` is the code for eq.4 in the paper. 
But why v_lambda is the output of vnet in `v_lambda = vnet(cost_v.data)` ??

I'm looking forward to your reply. Thank you so much.

(If you can speak Chinese, could you reply me in Chinese? My English is not very good.)

**Comment by GGchen1997** [comment]
Thanks for your interest in our work!

We design a two-stage weight generation scheme in our paper. In the first stage, we generate the weight from the vnet, which serves as the initial value of the second stage weight. 

English may be more accurate.

Best,
Can

**Comment by GGchen1997** [comment]
This question has been answered.
